<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="ir000781" name="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest" tests="27" failures="0" timestamp="19 Jun 2015 14:33:06 GMT" time="2.239" errors="13">
  <testcase name="required_spec208_mustBePreparedToReceiveOnNextSignalsAfterHavingCalledSubscriptionCancel" time="0.303" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec213_failingOnSignalInvocation" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec213_failingOnSignalInvocation(SubscriberWhiteboxVerification.java:345)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec316_requestMustNotThrowExceptionAndMustOnErrorTheSubscriber(SubscriberWhiteboxVerification.java:466)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec311_requestMaySynchronouslyCallOnCompleteOrOnError(SubscriberWhiteboxVerification.java:448)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec315_cancelMustNotThrowExceptionAndMustSignalOnError(SubscriberWhiteboxVerification.java:460)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec301_mustNotBeCalledOutsideSubscriberContext" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec301_mustNotBeCalledOutsideSubscriberContext(SubscriberWhiteboxVerification.java:418)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec211_mustMakeSureThatAllCallsOnItsMethodsHappenBeforeTheProcessingOfTheRespectiveEvents(SubscriberWhiteboxVerification.java:333)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithPrecedingRequestCall" time="0.002" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_spec213_onSubscribe_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec202_shouldAsynchronouslyDispatch" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec202_shouldAsynchronouslyDispatch(SubscriberWhiteboxVerification.java:115)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec205_mustCallSubscriptionCancelIfItAlreadyHasAnSubscriptionAndReceivesAnotherOnSubscribeSignal" time="0.307" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec310_requestMaySynchronouslyCallOnNextOnSubscriber(SubscriberWhiteboxVerification.java:442)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithoutPrecedingRequestCall" time="0.302" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_spec213_onError_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.002" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec207_mustEnsureAllCallsOnItsSubscriptionTakePlaceFromTheSameThreadOrTakeCareOfSynchronization(SubscriberWhiteboxVerification.java:246)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec213_onNext_mustThrowNullPointerExceptionWhenParametersAreNull" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnError" time="0.002" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_spec203_mustNotCallMethodsOnSubscriptionOrPublisherInOnComplete" time="0.002" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_exerciseWhiteboxHappyPath" time="0.395" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec212_mustNotCallOnSubscribeMoreThanOnceBasedOnObjectEquality_specViolation(SubscriberWhiteboxVerification.java:339)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec209_mustBePreparedToReceiveAnOnCompleteSignalWithPrecedingRequestCall" time="0.304" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid" time="0.000" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec206_mustCallSubscriptionCancelIfItIsNoLongerValid(SubscriberWhiteboxVerification.java:240)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec308_requestMustRegisterGivenNumberElementsToBeProduced" time="0.610" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="required_spec201_mustSignalDemandViaSubscriptionRequest" time="0.003" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec204_mustConsiderTheSubscriptionAsCancelledInAfterRecievingOnCompleteOrOnError(SubscriberWhiteboxVerification.java:201)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
  <testcase name="required_spec210_mustBePreparedToReceiveAnOnErrorSignalWithoutPrecedingRequestCall" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest"/>
  <testcase name="untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists" time="0.001" classname="com.aol.simple.react.reactivestreams.TckWhiteBoxSubscriberTest">
    <error type="org.testng.SkipException" message="Not verified using this TCK.">
      <![CDATA[org.testng.SkipException: Not verified using this TCK.
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.notVerified(SubscriberWhiteboxVerification.java:779)
	at org.reactivestreams.tck.SubscriberWhiteboxVerification.untested_spec314_cancelMayCauseThePublisherToShutdownIfNoOtherSubscriptionExists(SubscriberWhiteboxVerification.java:454)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:497)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:74)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:673)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:846)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1170)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.runWorkers(TestRunner.java:1147)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:317)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:312)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:274)
	at org.testng.SuiteRunner.run(SuiteRunner.java:223)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1039)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:964)
	at org.testng.TestNG.run(TestNG.java:900)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:110)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:205)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:174)
]]>
    </error>
  </testcase>
</testsuite>
